addons/website/views/snippets/s_banner.xml

```diff
@@ -6,7 +6,7 @@
         <div class="container">
             <div class="row s_nb_column_fixed o_grid_mode">
                 <div class="justify-content-center o_grid_item g-height-4 g-col-lg-5 col-lg-5" data-name="Box" style="z-index: 1; grid-area: 2 / 1 / 10 / 6;">
-                    <h1><font style="font-size: 62px;">Sell Online. <strong>Easily.</strong></font></h1>
```

Comment:
Simple tests about this to see if it works, the first two things I tested did not work entirely:

1.
```html
<h1><span style="font-size: 62px;">Hello World</span></h1>
```
Select all, change font-style to a hx -> good `<hx>Hello World</hx>`. Undo. Does not go back to what it was. Undo again, ok. So two undo for no reason apparently.

2.
```html
<h1><span style="font-size: 62px;">Hello</span> World</h1>
```
Select all, change hx tag =>
```html
<hx><span >Hello World</span></hx>
```